recipes/tip/bld.bat

```diff
@@ -0,0 +1,19 @@
+setlocal EnableDelayedExpansion
+
+mkdir build
+cd build
+
+:: USER_VERSION is used to hardcode version in built executables.
+:: It is typically unnecessary because CMake configure-time uses
+:: git ops to discover tag or commit SHA, both of which are not 
+:: available since packages are built from tarballs.
+cmake .. -GNinja -DCMAKE_BUILD_TYPE:STRING=Release ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -DCMAKE_PREFIX_PATH:PATH="%LIBRARY_PREFIX%" ^
+    -DCONDA_PREFIX:PATH="%CONDA_PREFIX%" ^
+    -DUSER_VERSION:STRING="v%PKG_VERSION%"
+
```

Comment:
```suggestion
    -DUSER_VERSION:STRING="v%PKG_VERSION%" ^
    -DBUILD_SHARED_LIBS=ON

```
This may help with accidental static linking on Windows?

Reply:
TIP build system has been updated to dynamically link always for conda builds. The suggestion on line 15 is unnecessary.